src/main/java/github/grace000/erversay/RouteHandlers/Redirect.java

```diff
@@ -0,0 +1,36 @@
+package github.grace000.erversay.RouteHandlers;
+
+import github.grace000.erversay.Request;
+import github.grace000.erversay.ResponseBuilder;
+
+import static github.grace000.erversay.Constants.Headers.REDIRECT_HEADER;
+import static github.grace000.erversay.Constants.StatusCodes.NOT_FOUND_STATUS;
+import static github.grace000.erversay.Constants.StatusCodes.REDIRECT_STATUS;
+
+public class Redirect implements RouteHandler {
+    private ResponseBuilder responseBuilder = new ResponseBuilder();
+
+    public enum AcceptedMethods {
+        GET
+    }
+
+    public boolean isMethodAllowed(String method) {
+        for (AcceptedMethods acceptedMethods : AcceptedMethods.values()) {
+            if (acceptedMethods.name().equals(method)) {
+                return true;
+            }
```

Comment:
You don't need to explicitly return `true` here. `acceptedMethods.name().equals(method)` returns a boolean